src/NormalizingFlows.jl

```diff
@@ -1,5 +1,72 @@
 module NormalizingFlows
 
-# Write your package code here.
+using Bijectors
+using Optimisers
+using LinearAlgebra, Random, Distributions, StatsBase
+using ProgressMeter
+using ADTypes, DiffResults
+using Zygote, ForwardDiff, ReverseDiff, Enzyme
+
+using DocStringExtensions
+
+export NF, elbo, loglikelihood
+
+"""
+    NF(rng::AbstractRNG, vo, flow, args...; kwargs...)
+    NF(vo, flow, args...; kwargs...) = NF(Random.default_rng(), vo, flow, args...; kwargs...)
```

Comment:
```suggestion
    NF([rng::AbstractRNG, ]vo, flow, args...; kwargs...)
```
is the style for optional args

Reply:
https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation